core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -172,8 +172,6 @@ var TopicsAndSkillsDashboardPage = function() {
       'Assign skill to topic button', assignSkillToTopicButtons.first());
```

Comment:
Above, can you add a waitFor statement before the line `expect(await skillEditOptions.count()).toEqual(1);`? Ditto for `expect(await assignSkillToTopicButtons.count()).toEqual(1);`